src/test/java/com/yahoo/validatar/AppTest.java

```diff
@@ -100,15 +102,17 @@ public void testRunTests() throws Exception {
         App.run(emptyTest, parameterMap, parseManager, engineManager, formatManager);
     }
 
-    @Test(expectedExceptions = {RuntimeException.class})
+    @Test
     public void testParameterParsingFailure() throws IOException {
         System.setOut(new PrintStream(new FileOutputStream("target/out")));
         System.setErr(new PrintStream(new FileOutputStream("target/err")));
 
         String args[] = {"--parameter", "DATE:20140807"};
-        OptionSet options = App.parse(args);
-        App.splitParameters(options, "parameter");
-
+        try {
+            OptionSet options = App.parse(args);
+            Assert.fail("Should have thrown an Exception");
+        } catch (RuntimeException re) {
```

Comment:
This was done to fix the problem where the system printstreams re-set was never being reached?